content/docs/guides/contribute/conventions/_index.en.md

```diff
@@ -0,0 +1,23 @@
+---
+title: "Style guide"
+linkTitle: "Style guide"
+weight: 40
+description: "Coding style guide and best practices"
+---
+
+OSRD application is split in multiple services written in several languages. We try to follow general code best practices and follow specificity for each languages when required.
+
+## General rules
+
+- Explain what you're doing and why.
+- Document new code with doc comments.
+- Include clear, simple tests.
+- Break work into digestible chunks.
+- Take the time to pick good names.
+  Avoid non well-known abbreviations.
+- **Control and consistency over 3rd party code reuse**: Only add a dependency if it is absolutely necessary.
+  Every dependency we add decreases our autonomy and consistency.
+- **Don't re-invent every wheel**: As a counter to the previous point, don't re-invent everything at all costs.
```

Comment:
Based on [this transaltion](https://www.linguee.fr/francais-anglais/traduction/r%C3%A9inventer+la+roue.html)
```suggestion
- **Don't reinvent the wheel**: As a counter to the previous point, don't reinvent everything at all costs.
```